assets/svg/date-range.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="20px" viewBox="0 0 18 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>date-range</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="date-range" fill="#5B5B61">
+            <path d="M6,9 L4,9 L4,11 L6,11 L6,9 Z M10,9 L8,9 L8,11 L10,11 L10,9 Z M14,9 L12,9 L12,11 L14,11 L14,9 Z M16,2 L15,2 L15,0 L13,0 L13,2 L5,2 L5,0 L3,0 L3,2 L2,2 C0.89,2 0.01,2.9 0.01,4 L0,18 C0,19.1 0.89,20 2,20 L16,20 C17.1,20 18,19.1 18,18 L18,4 C18,2.9 17.1,2 16,2 Z M16,18 L2,18 L2,7 L16,7 L16,18 Z" id="Shape"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
There is a bit of unnecessary structure here but it looks like most of this is already being cleaned up by SVGO during import

![image](https://user-images.githubusercontent.com/1621608/99793088-c6488c80-2b30-11eb-98c1-f1bec6f1aa42.png)

However, we should change the `fill` to `fill="currentColor"` to inherit the current `color` rather than hardcoding it. Then just make sure the `color` for that is set correctly via CSS, if it isn't already.

Reply:
Updated.